src/main/java/io/airbyte/testingtool/argument_parser/ArgumentParser.java

```diff
@@ -0,0 +1,32 @@
+package io.airbyte.testingtool.argument_parser;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class ArgumentParser {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ArgumentParser.class);
+    private static final String PARSING_ARGUMENT_CHAR = "=";
+    private static final String COMMAND_START_CHAR = "/";
+
+    public static Map<String, String> argumentParser(String[] args) {
+        return Stream.of(args).collect(Collectors.toMap(ArgumentParser::getArgumentKey, ArgumentParser::getArgumentValue));
+    }
+
+    private static String getArgumentKey(String arg) {
+        return arg.startsWith(COMMAND_START_CHAR) ? arg : splitAndGetArgumentPart(arg, true);
```

Comment:
I would prefer a key value for command start like a constant.
For example, "/run-help : /run-help" => "some_constant : run-help"
In addition, I think we should list all such commands by an enum.

Reply:
updated it